wallet/wallet.go

```diff
@@ -382,9 +382,14 @@ func (w *KeystoreWallet) ExportKey(_ context.Context, addr address.Address) (*Ke
 	w.mu.Lock()
 	defer w.mu.Unlock()
 
-	key, exists := w.keys[addr]
```

Comment:
You could still check the cache and if it's not in there check the keystore? Not a big performance issue though